widget/embedded/src/constants/settings.tsx

```diff
@@ -0,0 +1,44 @@
+import { i18n } from '@lingui/core';
+import {
+  AutoThemeIcon,
+  DarkModeIcon,
+  LightModeIcon,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+
+export enum Mode {
+  DARK = 'dark',
+  LIGHT = 'light',
+  AUTO = 'auto',
+}
+
+export const themesList = [
```

Comment:
It seems this code is in wrong file as it is not constant

Reply:
I moved it to the setting page